lib/jazzy/source_declaration.rb

```diff
@@ -13,6 +14,20 @@ def render_as_page?
       children.any?
     end
 
+    def language
+      type.swift_type? ? 'Swift' : 'Objective-C'
+    end
+
+    def swift
+      type.swift_type?
+    end
+
+    # Magic strings for use by the specific syntax highlighting library
+    # Might be better to have this mapping take place in Highlighter.rb
```

Comment:
I agree, could have Highlighter::SWIFT etc. constants and `Highlighter.highlight_swift(source)` helpers to make the caller simpler.